src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/FormattingContext.cs

```diff
@@ -309,15 +312,14 @@ private void ValidateComponents(RazorCodeDocument oldCodeDocument, RazorCodeDocu
                 // position now contains the first non-whitespace character or 0. Get the corresponding FormattingSpan.
                 if (result.TryGetFormattingSpan(nonWsPos.Value, out var span))
                 {
-                    indentations[i] = new IndentationContext
+                    indentations[i] = new IndentationContext(firstSpan: span)
```

Comment:
Since this was the only nullable property on this class, I just moved it to the constructor. Again, required properties in C# 10 should fix this, and allow this change to be reverted.